apps/api/src/Greencheck/Logger.php

```diff
@@ -100,6 +100,10 @@ public function logResult(SitecheckResult $result)
         $latest = new LatestResult();
         $latest->setResult($result);
 
+        // TODO - see about using this logger of the one in the
+        // greencheck library
+        $this->redis->set("domains:$checkedUrl", json_encode($latest));
```

Comment:
this will only be called if a result is gotten from the greencheck lib in a sync way by waiting on the results. So i guess it's fine, but probably not really needed if the greencheck lib logger is also logging it. 

Which reminds me, there is a way to disable the logging of the greencheck lib, we should check it does store the result in redis.

Reply:
As far I can tell, the only code using this is the `GreencheckLogProcessor` and I'm not sure of the difference between that and the `GreencheckProcessor`.

The GreencheckProcessor  in this branch uses the TGWF\Sitecheck\Logger instead of that one in API at present, but I can change to use App\GreenCheck\Logger instead if it make more sense.